pouchdb-core/pouchdb-core.d.ts

```diff
@@ -245,63 +537,177 @@ declare namespace PouchDB {
                 RemoteDatabaseConfiguration;
     }
 
+    /** @todo: remove nodejs dep. */
+    interface EventEmitter extends NodeJS.EventEmitter {
 
+    }
 
-    interface Static {
+    interface Static extends EventEmitter {
         plugin(plugin: Plugin): Static;
 
+        version: string;
+
+        on(event: 'created', listener: (dbName: string) => any): this;
+        on(event: 'destroyed', listener: (dbName: string) => any): this;
+
+        /**
+         * Replicate data from source to target. Both the source and target can be a PouchDB instance or a string 
+         * representing a CouchDB database URL or the name of a local PouchDB database. If options.live is true, 
+         * then this will track future changes and also replicate them automatically. 
+         * This method returns an object with the method cancel(), which you call if you want to cancel live replication.
+         */
+        replicate<Content>(
+            source: string | Database<Content>, 
+            target: string | Database<Content>, 
+            options?: Core.ReplicateOptions,
+            callback?: Core.Callback<any, Core.ReplicationResultComplete<Content>>
+        ): Core.Replication<Content>;
+
+        /** 
+         * Sync data from src to target and target to src. This is a convenience method for bidirectional data replication.
+         * 
+         * In other words, this code:
+         * `PouchDB.replicate('mydb', 'http://localhost:5984/mydb')`;
+         * `PouchDB.replicate('http://localhost:5984/mydb', 'mydb')`;
+         * is equivalent to this code:
+         * `PouchDB.sync('mydb', 'http://localhost:5984/mydb')`;
+         */
+        sync<Content>(
+            source: string | Database<Content>, 
+            target: string | Database<Content>, 
+            options?: Core.SyncOptions,
+            callback?: Core.Callback<any, Core.SyncResultComplete<Content>>
+        ): Core.Sync<Content>;
+
         new<Content extends Core.Encodable>(name?: string,
             options?: Configuration.DatabaseConfiguration): Database<Content>;
+
+        /**
+         * The returned object is a constructor function that works the same as PouchDB, 
+         * except that whenever you invoke it (e.g. with new), the given options will be passed in by default.
+         */
+        defaults(options: Configuration.DatabaseConfiguration): {
+            new<Content extends Core.Encodable>(name?: string,
+                options?: Configuration.DatabaseConfiguration): Database<Content>;
+        }
     }
 
     interface Database<Content extends Core.Encodable>  {
+
+        replicate: {
```

Comment:
This is only available when using `pouchdb-replication`, so it'd be best to put this in a separate file+folder.